sql/mozfun/datetime_util/braze_parse_time/udf.sql

```diff
@@ -0,0 +1,12 @@
+CREATE OR REPLACE FUNCTION datetime_util.braze_parse_time(json_str STRING)
+RETURNS STRING
+LANGUAGE js
+AS
+  """
+  try {
+    var json = JSON.parse(json_str);
+    return json['$time'];
+  } catch (e) {
+    return null;
+  }
+""";
```

Comment:
Could you add a few udf tests below? Here's an example: 
https://github.com/mozilla/bigquery-etl/blob/main/sql/mozfun/json/from_map/udf.sql#L27-L37